usgs-details.html

```diff
@@ -34,6 +34,59 @@
         var BASE_URL = 'https://waterservices.usgs.gov/nwis/iv/';
         var REQUEST_URL = BASE_URL + '?format=json' + '&sites=' + SITE_ID;
 
+        var METRICS = {
+            temperature: {
+                code: '00010',
+                units: '°F',
+                transform: function(c) {
+                    var f = c * 9 / 5 + 32;
+                    return Math.round(f * 10) / 10;
+                },
+                fallback: [
+                    42, 32, 42, 51, 61, 71, 80, 90, 80, 71, 61, 51
+                ]
+            },
+            ph: {
+                code: '00400',
+                units: '',
+                transform: function(ph) { return ph; }, // Identity
+                fallback: [
+                    5.5, 5, 5.5, 6, 6.5, 7, 7.5, 8, 7.5, 7, 6.5, 6
+                ],
+            },
+            dissolvedOxygen: {
+                code: '00300',
+                units: 'mg/L',
+                transform: function(dO) { return dO; }, // Identity
+                fallback: [
+                    10, 12, 10, 8, 6, 4, 2, 1, 2, 4, 6, 8
+                ]
+            },
+            salinity: {
+                code: '00095',
+                units: 'ppm',
+                transform: function(uScm) {
+                    var ppm = uScm / 1.56;
+                    return Math.round(ppm);
```

Comment:
The variable names document that the value is being converted from μS/cm to PPM. Without it the division by `1.56` would need a comment, I thought this to be a better approach.